.github/automerge.yml

```diff
@@ -0,0 +1,21 @@
+name: Auto-merge
+
+on:
+  workflow_run:
+    types:
+      - completed
+    workflows:
+      - 'Lucky App CI'
```

Comment:
misnomer?

Reply:
Ooh, yea needs to be be the main ci workflow